include/seqan3/search/configuration/max_error.hpp

```diff
@@ -67,26 +56,21 @@ class max_error : public pipeable_config_element<max_error<errors_t...>, std::ar
         }
     }
 
-    static_assert(check_consistency(errors_t{}...),
-                  "You may not use the same error specifier more than once.");
-
 public:
-
     //!\privatesection
     //!\brief Internal id to check for consistent configuration settings.
     static constexpr detail::search_config_id id{detail::search_config_id::max_error};
 
     //!\publicsection
     /*!\name Constructor, destructor and assignment
-     * \brief Defaulted all standard constructor.
      * \{
      */
-    constexpr max_error()                              noexcept = default; //!< Default constructor.
-    constexpr max_error(max_error const &)             noexcept = default; //!< Copy constructor.
-    constexpr max_error(max_error &&)                  noexcept = default; //!< Move constructor.
-    constexpr max_error & operator=(max_error const &) noexcept = default; //!< Copy assignment.
-    constexpr max_error & operator=(max_error &&)      noexcept = default; //!< Move assignment.
-    ~max_error()                                       noexcept = default; //!< Destructor.
+    constexpr max_error() = default; //!< Defaulted.
+    constexpr max_error(max_error const &) = default; //!< Defaulted.
+    constexpr max_error(max_error &&) = default; //!< Defaulted.
+    constexpr max_error & operator=(max_error const &) = default; //!< Defaulted.
+    constexpr max_error & operator=(max_error &&) = default; //!< Defaulted.
+    ~max_error() = default; //!< Defaulted
```

Comment:
Can you explain to me, why you changed the descriptions?
Why not say 'Defaulted move constructor'?

Reply:
We agreed on this a long while ago, because this documentation is really only for avoiding doxygen warnings (since doxygen shows the `= default` in the docu) so we will just consistently use ``//!< Defaulted.` everywhere when constructors/assignment are explicitily defaulted :)

Reply:
can you please add `noexcept` for move construction and move assignment as well as the destructor.

Reply:
It would be weird to state some of them but not all, indicating that the other might actually not be noexcept. So either we put noexcept everywhere explicitly or let it be done implicitly by the compiler.